config/locales/es.yml

```diff
@@ -102,6 +107,8 @@ es:
   passed: Todo en orden
   passed_with_warnings: Funcionó, pero hay cosas para mejorar
   pending: pendiente
+  pending_messages: Tenés mensajes sin leer
+  pending_messages_explanation: No vas a poder enviar nuevas soluciones hasta no leer los mensajes pendientes
```

Comment:
Perhaps a slightly modified version of the last one, that makes explicit that the teacher wants your attention: _¡Tu docente te dejó un mensaje! Revisalo antes de seguir enviando soluciones._